pkg/apis/app/v2/kieapp_types.go

```diff
@@ -154,18 +154,31 @@ type KieServerSet struct {
 	// When set to true will configure the KIE Server image to disable all capabilities that are not related to decisions, Default to true for RHDM environments and false to RHPAM.
 	DecisionsOnly *bool `json:"decisionsOnly,omitempty"`
 	// Number of max KIE Executor sessions, it must be lower than the value of max-pool-size, by default is max-pool-size set to 60. Max pool size can be set by system property jboss.mdb.strict.max.pool.size (using javaOptsAppend: "-Djboss.mdb.strict.max.pool.size=40"), for more information see https://access.redhat.com/solutions/2955481.
-	KieExecutorMDBMaxSession *int `json:"kieExecutorMDBMaxSession,omitempty"`
+	KieExecutorMDBMaxSession *int              `json:"kieExecutorMDBMaxSession,omitempty"`
+	TerminationRoute         *TerminationRoute `json:"terminationRoute,omitempty"`
+}
+
+type TerminationRoute struct {
+	// Enable edge termination
+	EnableEdge bool `json:"enableEdge,omitempty"`
```

Comment:
@desmax74 you mentioned that this is only supported for http, right? We should mentioned it in description as well. And probably also that it is required to have set disableSsl to true as well.

Reply:
ok